rom/dev/src/main.rs

```diff
@@ -110,6 +110,10 @@ pub extern "C" fn rom_entry() -> ! {
         }
     );
 
+    // Set the ROM version
+    let rom_info = unsafe { &CALIPTRA_ROM_INFO };
+    env.soc_ifc.set_rom_fw_rev_id(rom_info.version);
```

Comment:
Can we add a test in rom/tests (choose the appropriate module in that) to test that the version is set correctly?

Reply:
This is a good point. This test is missing in ROM. I'll add it.

Reply:
This has been added now